StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/STPAnalyticsClient+PaymentSheet.swift

```diff
@@ -414,6 +414,8 @@ extension PaymentSheet.Configuration {
         payload["appearance"] = appearance.analyticPayload
         payload["billing_details_collection_configuration"] = billingDetailsCollectionConfiguration.analyticPayload
         payload["preferred_networks"] = preferredNetworks?.map({ STPCardBrandUtilities.apiValue(from: $0) }).joined(separator: ", ")
+        payload["ui_mode"] = paymentMethodLayout.description
```

Comment:
Why UI mode?  I'd expect this to be called "payment_method_layout" mirroring the API

Reply:
Got it here, I can change to `payment_method_layout`.
![CleanShot 2024-06-13 at 08 39 42](https://github.com/stripe/stripe-ios/assets/88012362/f7835431-1a74-41f3-ba22-7d295d965863)

Reply:
Ah I probably wrote that before we knew the API.